dotnet-engine/Yggdrasil.Benchmarks/Program.cs

```diff
@@ -0,0 +1,60 @@
+﻿using System;
+using System.Text.Json;
+using BenchmarkDotNet.Attributes;
+using BenchmarkDotNet.Running;
+using BenchmarkDotNet.Configs;
+using BenchmarkDotNet.Columns;
+using BenchmarkDotNet.Running;
+using Newtonsoft.Json.Linq;
+using Yggdrasil;
+
+[Config(typeof(Config))]
+public class YggBench
+{
+    private YggdrasilEngine yggdrasilEngine;
+
+    private class Config : ManualConfig
+    {
+        public Config()
+        {
+            AddColumn(BenchmarkDotNet.Columns.StatisticColumn.OperationsPerSecond);
```

Comment:
Not strictly necessary but gives us the output we want to be able to compare against the others